templates/zerver/help/mute-a-stream.md

```diff
@@ -0,0 +1,23 @@
+### Mute a stream
+
+Muting streams to reduce notification clutter is easy. Simply hover your mouse
+over the stream in the streams list on the left of the chat, press the down
+chevron button
+
+![streams arrow](static/images/help/streams-arrow.png)
+
+to bring up the stream popover menu and press **Mute the stream**. The muted
```

Comment:
The chevron image should be inline, so you can delete the blank lines 6 and 8 for a continuous sentence. AKA `press the down chevron button (![chevron](static/images/help/chevron.png))  to bring up` (the inline icon images have parentheses surrounding them)

Also, we usually don't use "popover" in our docs; instead we use "drop down", so you should replace "popover" with "dropdown"